examples/python/download_dashboard_pdf.py

```diff
@@ -59,10 +61,10 @@ def download_dashboard(
     width: int = 545,
     height: int = 842,
     filters: Optional[Dict[str, str]] = None,
-):
+    ):
+
     """Download specified dashboard as PDF"""
-    assert dashboard.id
-    id = int(dashboard.id)
+    id = dashboard.id
```

Comment:
New issue here ... API 4.0 (SDK) requires sdk.create_dashboard_render_task(dashboard_id: str) (it was int according to this script)